dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -683,15 +683,15 @@ BEGIN
     PUSHBUTTON "S&chimbare pictogramă…", IDC_EDITTYPE_CHANGE_ICON, 150, 10, 70, 14
     LTEXT "&Acțiuni:", -1, 5, 37, 70, 10
     LISTBOX IDC_EDITTYPE_LISTBOX, 5, 50, 140, 60, LBS_NOINTEGRALHEIGHT | LBS_HASSTRINGS | LBS_OWNERDRAWFIXED | LBS_NOTIFY | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "N&ou…", IDC_EDITTYPE_NEW, 150, 50, 70, 14
+    PUSHBUTTON "No&u", IDC_EDITTYPE_NEW, 150, 50, 70, 14
```

Comment:
"No&u" does collide with AUTOCHECKBOX "Ca&ută
a few lines below. One of both must be changed.

Reply:
Resolved, he changed to  AUTOCHECKBOX "Căuta&re